packages/rocketchat-lib/lib/slashCommand.js

```diff
@@ -3,14 +3,16 @@ RocketChat.slashCommands = {
 };
 
 RocketChat.slashCommands.add = function(command, callback, options = {}, result) {
-	RocketChat.slashCommands.commands[command] = {
-		command,
-		callback,
-		params: options.params,
-		description: options.description,
-		clientOnly: options.clientOnly || false,
-		result
-	};
+	if (!options.condition || options.condition === null || (typeof options.condition === 'function' && options.condition() === true)) {
```

Comment:
is this `condition` option being used?

Reply:
I'll remove that. It was implemented for provides add control for slash commands, but I went back from idea of the your usage.